docs/DeveloperGuide.md

```diff
@@ -281,25 +281,36 @@ We categorise our user stories into four main epics:
+| S12 | `LOW`    | user with many students stored in **TAB**  | sort students by tag                       | locate a student easily                                                |
+| L1  | `HIGH`   | user                                       | schedule recurring lessons for a student   | keep track of when the student's lessons are                           |
+| L2  | `HIGH`   | user                                       | delete a student's lesson                  | remove lessons that I no longer have                                   |
+| L3  | `MEDIUM` | user                                       | edit a student's lesson                    | update lesson information                                              |
+| L4  | `MEDIUM` | user                                       | add makeup lessons for a student           | differentiate between the two different lessons                        |
+| L5  | `MEDIUM` | user                                       | view the lessons I have for the week       | easily manage my schedule                                              |
+| L6  | `MEDIUM` | user                                       | view a calendar of my scheduled lessons    | plan ahead while managing my schedule                                  |
+| U1  | `HIGH`   | new user                                   | see usage instructions                     | refer to instructions when I forget how to use **TAB**                 |
+| U2  | `HIGH`   | new user                                   | purge all current data                     | remove all sample/experimental data I used while exploring **TAB**     |
+| U3  | `HIGH`   | user                                       | find a student by name                     | locate details of persons without having to go through the entire list |
+| U4  | `HIGH`   | user                                       | undo accidental changes                    |                                                                        |
+| U5  | `MEDIUM` | user                                       | filter students by tag                     | view a list of all the students that feature a specific tag            |
+| U6  | `MEDIUM` | user with incomplete information           | skip less important information to add     | update later when I have the rest of the information                   |
+| U7  | `MEDIUM` | user                                       | select and copy data to desktop clipboard  | paste and use the data outside of **TAB**                              |
+| U8  | `LOW`    | user who uses many tags                    | view all tags                              | remember my tags easily                                                |
+| U9  | `LOW`    | expert user                                | customise the commands and input fields    | be not limited by a template                                           |
```

Comment:
Minor comment; I think "so that I can _not be_ limited by a template" might sound better?